proxy/grpcproxy/cache/store.go

```diff
@@ -73,7 +77,11 @@ func (c *cache) Add(req *pb.RangeRequest, resp *pb.RangeResponse) {
 	defer c.mu.Unlock()
 
 	if req.Revision > c.compactedRev {
-		c.lru.Add(key, resp)
+		if req.Revision == 0 {
+			c.lru.Set(key, resp, defaultCurrentTTL)
+		} else {
+			c.lru.Set(key, resp, defaultHistoricTTL)
+		}
```

Comment:
is it possible the req.Revision from the req is the latest revision, eg kv store revision = 100 and req.Revision ==100? then Keys on that revision is the latest and are subject to change? if that's possible, does it means that we need to use defaultCurrentTTL then?